src/core/src/bootstrap/EnvLayer.py

```diff
@@ -428,6 +429,25 @@ def write_with_retry_using_temp_file(file_path, data, mode='w'):
                     else:
                         raise Exception("Unable to write to {0} (retries exhausted). Error: {1}.".format(str(file_path), repr(error)))
 
+        @staticmethod
+        def delete_artifacts_from_dir(dir_name, artifact_identifier, raise_if_delete_failed=False):
```

Comment:
Nit: avoid using the non-standard word 'artifacts' as relates to a basic file system operation. At its simplest, it's just a 'del_files' operation that operates on a specific directory (inferable from the function signature). Is that fair? Nomenclature-wise, suggest removing artifacts across the code-base as relates to this body of functionality.

Reply:
Resolved